exetera/core/fields.py

```diff
@@ -55,11 +52,15 @@ def __bool__(self):
         #   if f is not None:
         return True
 
-    # def __len__(self):
-    #     raise NotImplementedError()
-    #
-    # def get_spans(self):
-    #     raise NotImplementedError("Please use get_spans() on specific fields, not the field base class.")
+    def get_spans(self):
```

Comment:
I note it was already missing but can you also add the license boilerplate here?